src/main/java/IO/DbWriter.java

```diff
@@ -0,0 +1,104 @@
+package IO;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.nio.charset.Charset;
+import java.util.Random;
+
+import data.Database;
+
+/* DbWriter class */
+public class DbWriter {
+
+  private String runId;
+  private String type;
+  private Database database;
+  private String fileName;
+  private static final int hashLength = 10;
+  private OutputStream out;
+  // private static final String extension = ".bin";
```

Comment:
the .bin usually stands for executable binary, you can use another extension e.g. recordio, or not use any extension.